ledger/simulation/tracer.go

```diff
@@ -140,6 +140,12 @@ func (tracer *evalTracer) BeforeTxnGroup(ep *logic.EvalParams) {
 		tracer.result.TxnGroups[0].AppBudgetAdded = uint64(*ep.PooledApplicationBudget)
 	}
 
+	// Override transaction group budget if specified in request, retrieve from tracer.result
+	if tracer.result.EvalOverrides.ExtraAppBudget != nil && ep.PooledApplicationBudget != nil {
```

Comment:
I'm only now realizing this allows extra budget in apps but not logicsigs.  Do we want that?

Reply:
It would be interesting, but not very commonly used I guess?

Reply:
I assume that with this PR as it is now, one could, in a simulated transaction group, create an wrapper app that incorporates the logicsig code, then call into it that app, with elevated opcode budget? i.e. there's nothing a logicsig can do that an app can't do.